articles/energy-data-services/osdu-services-on-adme.md

```diff
@@ -37,7 +37,7 @@ Azure Data Manager for Energy is currently compliant with the M23 OSDU® milesto
 
 ### Ingestion services
 - **EDS DMS [[Preview]](how-to-enable-external-data-sources.md)**: Pulls specified data (metadata) from OSDU-compliant data sources via scheduled jobs while leaving associated dataset files (LAS, SEG-Y, etc.) stored at the external source for retrieval on demand.
-  - **EDS Fetch & Ingest DAG**: Facilitates fetching data from external providers and ingesting it into the OSDU platform. It involves steps like registering with providers, creating data jobs, and triggering ingestion.
+  - **EDS Fetch & Ingest DAG**: Facilitates fetching data from external providers and ingesting it into the OSDU platform. It involves steps like registering with providers, creating data jobs, and triggering ingestion.With M23 release EDS Fetch and Ingest dag is enabled with new features like Parent Data mapping and Reference data mapping.
```

Comment:
Nit: 

With the M23 release, EDS Fetch and Ingest DAG includes new features like Parent and Reference data mapping.

Reply:
Also a space after the period. So it will be:

   - **EDS Fetch & Ingest DAG**: Facilitates fetching data from external providers and ingesting it into the OSDU platform. It involves steps like registering with providers, creating data jobs, and triggering ingestion. With the M23 release, EDS Fetch and Ingest DAG includes new features like Parent and Reference data mapping.

Reply:
Done.